packages/ckeditor5-mention/src/mention.ts

```diff
@@ -214,16 +209,25 @@ export default class Mention extends Plugin {
  * 				} );
  *			} );
  *		}
+ * ```
  *
- * @typedef {Object} module:mention/mention~MentionFeed
- * @property {String} [marker] The character which triggers autocompletion for mention. It must be a single character.
- * @property {Array.<module:mention/mention~MentionFeedItem>|Function} feed Autocomplete items. Provide an array for
+ * @property marker The character which triggers autocompletion for mention. It must be a single character.
+ * @property feed Autocomplete items. Provide an array for
  * a static configuration (the mention feature will show matching items automatically) or a function which returns an array of
  * matching items (directly, or via a promise). If a function is passed, it is executed in the context of the editor instance.
- * @property {Number} [minimumCharacters=0] Specifies after how many characters the autocomplete panel should be shown.
- * @property {Function} [itemRenderer] A function that renders a {@link module:mention/mention~MentionFeedItem}
+ * @property minimumCharacters=0 Specifies after how many characters the autocomplete panel should be shown.
+ * @property itemRenderer A function that renders a {@link module:mention/mention~MentionFeedItem}
  * to the autocomplete panel.
  */
+export type MentionFeed = {
+	marker: string;
+	feed?: Array<MentionFeedItem> | FeedCallback;
```

Comment:
I don't think it's optional. Especially if you have to use non-null assertion in `MentionUI#init`.

Reply:
I typed it as optional, because in `mentionui.ts` it's not specified in the object.
![image](https://user-images.githubusercontent.com/114912428/212051518-11b911dc-f753-4dcb-bbd6-3d906c103fad.png)

If it makes sense to type it as not optional, I have to use casting to resolve compiler errors:
![image](https://user-images.githubusercontent.com/114912428/212051762-dd48c361-878d-4596-91ad-d25d241513be.png)